apps/provider-console/next.config.js

```diff
@@ -31,8 +33,20 @@ const nextConfig = {
     });
     config.externals.push("pino-pretty");
     return config;
-  },
+  }
 };
 
-module.exports = nextConfig;
+// Sentry webpack plugin configuration
+const sentryWebpackPluginOptions = {
+  silent: true, // Suppresses all logs
+  dryRun: process.env.NODE_ENV !== 'production', // Only upload source maps in production
+  release: require("./package.json").version,
+  // Add Sentry auth token only for production builds
+  authToken: process.env.NODE_ENV === 'production' ? process.env.SENTRY_AUTH_TOKEN : undefined,
+};
```

Comment:
i realise this is a copypaste prob but the comments are not of much help generally )